src/helm/benchmark/static/schema.yaml

```diff
@@ -326,6 +326,13 @@ models:
     description: Codex-style model based on the [Codex paper](https://arxiv.org/pdf/2107.03374.pdf), but the details (including size) are unknown.
     creator_organization: OpenAI
     access: limited
+  - name: openai/chat-gpt
+    display_name: ChatGPT
+    description: Sibling model to InstructGPT which interacts in a conversational way. See [OpenAI's announcement](https://openai.com/blog/chatgpt/). The size of the model is unknown.
```

Comment:
Not clear it's a sibling model, so we just say a large language model that you interact with via conversation.

Reply:
This was from their blog post: https://openai.com/blog/chatgpt.